scripts/jenkins/gitlint.ini

```diff
@@ -8,7 +8,7 @@ line-length=80
 words=wip,WIP
 
 [title-match-regex]
-regex=^(?!SCRD).*\(((bsc#|SCRD-)[0-9]+(,\s)?)+\)$
+regex=^(?!SOC).*\(((bsc#|SOC-)[0-9]+(,\s)?)+\)$
```

Comment:
We need to support both SCRD & SOC at least for the short term no? Otherwise changes that are in flight may fail...

Reply:
There will be hard switch from SCRD to SOC. From this point on SCRD should not be used, even for backports we should then rename SCRD to SOC.

Reply:
So then we are going to break all existing Jira <=> Gerrit linkages? Or are we going to have to revert and re-submit patches that have already landed but haven't yet been promoted from staging to devel (or worse will we need to do taht for changes that have already made it into devel???) as regards the automated notification of changes being pulled into a Product release? Otherwise even if you flick the switch tomorrow you are going to have changes in flight that haven't been promoted from Staging to Devel that will be using SCRD based Jira IDs...